style.css

```diff
@@ -128,37 +146,99 @@
   padding: 10px 12px;
   margin-right: 12px;
   border: 1px solid #8993a4;
-  font-family: "Inter";
+  font-family: "Inter", sans-serif;
   font-style: normal;
   font-weight: 500;
   font-size: 15px;
   line-height: 20px;
   color: #091e42;
 }
 
+form button {
+  margin: 15px;
+  background: #ff6b00;
+  color: #fff;
+  border: none;
+  width: 131px;
+  height: 48px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  margin-bottom: 124px;
+  cursor: pointer;
+}
+
 .works button {
   border: none;
   width: 124px;
   background-color: #ff6b00;
-  font-family: "Inter";
+  font-family: "Inter", sans-serif;
   font-weight: 700;
   font-size: 17px;
   line-height: 24px;
-  color: #ffffff;
+  color: #fff;
   padding: 12px;
   margin-left: 24px;
   margin-top: 12px;
 }
 
+.art button {
+  border: none;
+  width: 327px;
+  text-align: center;
+  background-color: #ff6b00;
+  font-family: "Inter", sans-serif;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  color: #fff;
+  padding: 12px 12px;
+  margin-top: 22px;
+}
+
+.aboutMe button {
+  border: none;
+  width: 161px;
+  height: 46px;
+  text-align: center;
+  background-color: #ff6b00;
+  font-family: "Inter", sans-serif;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  color: #fff;
+  padding: 12px 12px;
+  margin-left: 24px;
+  margin-top: 22px;
+}
+
+form button:active {
+  color: #f4f1f0;
+  background-color: #e05304;
+}
+
+form button:disabled {
+  background-color: #f4f1f0;
+  color: #5b5957;
+}
+
+form button:hover {
+  border: 1px solid #f55800;
+  color: #f55800;
+  background: #fff5e1;
+}
+
```

Comment:
- [ ] Nice work adding `border` for hover interaction but Like I mentioned before Please add active (pressed) interaction by adding `:active` pseudo-class to the button with Figma design styling.

Figma Design:
![image](https://user-images.githubusercontent.com/68796232/160893923-9b6e32a9-0b86-46c9-a442-9d18f965ed6b.png)

Reply:
@Omar-Muhamad  I have added active at line num 217. Could you please let me know more about what you mean by the above implementation?

Reply:
Yes you added it but it's not working so please see what causing that and fix it.
You can test it by pressing the button and see if the colors changed.
When it work well with you ask for another review and it will be accepted